scripts/localize.sh

```diff
@@ -12,6 +12,17 @@
 # This script can be run with no arguments:
 #  ./localize.sh
 
+ENGLISH_ONLY=false
+
+for ARGUMENT in "$@"
+do
+    if [ "$ARGUMENT" = "ENGLISH_ONLY" ]; then
+        echo "Only updating English translations due to ENGLISH_ONLY flag"
+        ENGLISH_ONLY=true
+        break
+    fi
+done
```

Comment:
This can’t be the easiest way, but I also don’t know bash, so 🤷

Reply:
Maybe you want a positional argument, rather than all of them? `$1`?